examples/alanine_dipeptide_tps/AD_tps_2b_run_fixed.ipynb

```diff
@@ -33,9 +40,7 @@
   {
    "cell_type": "code",
    "execution_count": 2,
-   "metadata": {
-    "collapsed": false
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
     "old_storage = paths.Storage(\"tps_nc_files/alanine_dipeptide_fixed_tps_traj.nc\", \"r\")"
```

Comment:
this line should be in line with the final name used in `AD_tps_1b_trajectory` (with or without the directory, whatever you decide)